app/graphql/types/profile.rb

```diff
@@ -44,7 +45,39 @@ class Types::Profile < Types::BaseObject
     null: true,
     description: 'The message this user has submitted for the Hall of Fame'
 
+  field :location, String,
```

Comment:
`waifu_or_husbando` needs to document the standard expected values - `"Waifu"` and `"Husbando"`

Reply:
`pro_tier`: The Pro level the user currently has

Reply:
`pro_message`: The message this user has submitted to the Hall of Fame